### PR TITLE
xml-tooling-c 3.2.4

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  ==> /opt/homebrew/Cellar/shibboleth-sp/3.4.1/sbin/shibd -t
  configuration is invalid, check console or log for specific problems
  2023-06-13 11:55:10 CRIT XMLTooling.Config : libcurl lacks OpenSSL-specific options, this will greatly limit functionality
  2023-06-13 11:55:10 ERROR XMLTooling.ParserPool : fatal error on line 0, column 0, message: unable to open primary document entity '/opt/homebrew/Cellar/xml-tooling-c/3.2.3/share/xml/xmltooling/catalog.xml'
  2023-06-13 11:55:10 ERROR XMLTooling.ParserPool : catalog loader caught exception: XML error(s) during parsing, check log for specifics
  2023-06-13 11:55:10 WARN XMLTooling.ParserPool : warning on line 0, column 0, message: unable to open primary document entity '/opt/homebrew/Cellar/shibboleth-sp/3.4.1/share/xml/shibboleth/xenc-schema.xsd'
  2023-06-13 11:55:10 ERROR XMLTooling.ParserPool : error on line 96, column 73, message: namespace 'http://www.w3.org/2001/04/xmlenc#' is referenced without import declaration
  2023-06-13 11:55:10 ERROR XMLTooling.ParserPool : error on line 96, column 73, message: referenced element 'xenc:EncryptedData' not found
  2023-06-13 11:55:10 WARN XMLTooling.ParserPool : warning on line 0, column 0, message: unable to open primary document entity '/opt/homebrew/Cellar/shibboleth-sp/3.4.1/share/xml/shibboleth/xml.xsd'
  2023-06-13 11:55:10 WARN XMLTooling.ParserPool : warning on line 0, column 0, message: unable to open primary document entity '/opt/homebrew/Cellar/shibboleth-sp/3.4.1/share/xml/shibboleth/xmldsig-core-schema.xsd'
  2023-06-13 11:55:10 ERROR XMLTooling.ParserPool : fatal error on line 1, column 1, message: invalid document structure
  2023-06-13 11:55:10 ERROR XMLTooling.ParserPool : fatal error on line 14, column 103, message: fatal error during schema scan
  2023-06-13 11:55:10 ERROR Shibboleth.Config : error while loading resource (/opt/homebrew/etc/shibboleth/shibboleth2.xml): XML error(s) during parsing, check log for specifics
  2023-06-13 11:55:10 FATAL Shibboleth.Config : caught exception while loading configuration: XML error(s) during parsing, check log for specifics
```

**Comment by nandahkrishna** [comment]
Since there's no new version of `shibboleth-sp` at the moment, it just needs to be revision bumped. I've pushed a commit to do so.

File: Formula/shibboleth-sp.rb
Comment (carlocab) [inline Formula/shibboleth-sp.rb:1]: This is looking up a path inside `xml-tooling-c`'s keg via a Cellar path. We should fix that so it stops breaking on version/revision bumps.

Comment (nandahkrishna) [reply Formula/shibboleth-sp.rb:1]: > This is looking up a path inside `xml-tooling-c`'s keg via a Cellar path. We should fix that so it stops breaking on version/revision bumps.

Will try to work on this. Just wanted to note that `shibboleth-sp` looks up paths in `opensaml` as well.

In the past, some PRs have bumped versions of `shibboleth-sp` and these related formulae together (e.g. https://github.com/Homebrew/homebrew-core/pull/118868, https://github.com/Homebrew/homebrew-core/pull/67125). A revision bump was suggested in https://github.com/Homebrew/homebrew-core/pull/67125#issuecomment-748042225 and was done in https://github.com/Homebrew/homebrew-core/pull/34821. Just for context.

**Comment by carlocab** [comment]
Switched base branch to `openssl-migration-staging`.

**Comment by nandahkrishna** [comment]
> This is looking up a path inside `xml-tooling-c`'s keg via a Cellar path. We should fix that so it stops breaking on version/revision bumps.

I guess I'll deal with this in a future PR.

**Review by nandahkrishna — APPROVED** [review]

**Pull request merged**